src/workflows/foraging.bonsai

```diff
@@ -1324,7 +1324,7 @@
                       <harp:VisualIndicators>On</harp:VisualIndicators>
                       <harp:Heartbeat>Disabled</harp:Heartbeat>
                       <harp:IgnoreErrors>false</harp:IgnoreErrors>
-                      <harp:PortName>COM14</harp:PortName>
+                      <harp:PortName>COM25</harp:PortName>
```

Comment:
why is this changing?

Reply:
This is for my computer. It's dependent on Settings_box1.csv and will not affect running on the training rig.

Reply:
In this case I would recommend not adding bonsai files to git's staged changes when you commit, which is what I typically do.

Reply:
Sounds good!